mbuild/tests/test_utils.py

```diff
@@ -186,3 +187,325 @@ def test_has_ipython(self):
 
     def test_removed_func_error(self):
         RemovedFuncError("old_function", "new_function", "0.0.0", "0.1.0")
+
+
+class TestUtilsConversion(BaseTest):
+    def test_RB_to_OPLS_c5_not_0(self):
+        with pytest.raises(
+            ValueError,
+            match=r"ERROR: c5 must equal zero, so this conversion is not possible.",
+        ):
+            c0 = 0.1
+            c1 = 0.1
+            c2 = -0.2
+            c3 = -0.1
+            c4 = -0.2
+            c5 = 0.3
+            RB_to_OPLS(c0, c1, c2, c3, c4, c5)
+
+    def test_RB_to_OPLS_f0_not_0(self):
+        with pytest.raises(
+            ValueError,
+            match=r"ERROR: f0 \= 2 \* \(c0 \+ c1 \+ c2 \+ c3 \+ c4 \+ c5\) must equal zero, "
+            "so this conversion is not possible.",
+        ):
+            c0 = 0.4
+            c1 = 0.4
+            c2 = -0.1
+            c3 = 0.4
+            c4 = -0.2
+            c5 = 0
+            RB_to_OPLS(c0, c1, c2, c3, c4, c5)
+
+    def test_RB_to_OPLS_error_tol_not_float(self):
+        with pytest.raises(
+            TypeError,
+            match=r"ERROR: The error_tol variable must be a float.",
+        ):
+            c0 = 0.1
+            c1 = 0.1
+            c2 = -0.2
+            c3 = -0.1
+            c4 = -0.2
+            c5 = 0.3
+            RB_to_OPLS(c0, c1, c2, c3, c4, c5, error_tol="s")
```

Comment:
Im guessing that the `c5` check occurs later than the `error_tol` check, but for the sake of this test, lets make `c5=0.0` so we can be sure that we are only testing for one error.

Reply:
Fixed. c5=0.0